README.md

```diff
@@ -1,5 +1,89 @@
-# ninai/pipeline
-Processing pipeline for scans and behavioral files.
+# Odor Meso
 
-# Docker container
-* [ninai/pipeline](https://hub.docker.com/r/ninai/pipeline/) `ninai/pipeline` 
++ DataJoint pipeline to capture the data for the mice two-photon imaging experiments in the presence of olfactory stimulation.
+
++ This is a cloned of the [cajal/pipeline](https://github.com/cajal/pipeline) with modifications. 
+
+---
+## Install pipeline
+
+### Step 1 - Clone the repositories
+
++ Launch a new terminal and run the following
+    ```
+    cd ~/<Enter custom path>
+    git clone https://github.com/reimerlab/odor_meso
+    git clone --branch release-1.0 https://github.com/ppfaff/DataPlot
+    cd odor_meso
+    ```
+
+### Step 2 - Create `.env`
+
++ At the root of the `odor_meso` repository folder, create a new file `.env` from the `env_template`.
+
++ Specify the database's `DJ_HOST`, `DJ_USER`, and `DJ_PASS`.
+
+### Step 3 - Create `paths.init` and `uinit.init`
+
++ This file is required for the `DataMan` and `plot_dm` functions.
+
++ At the root of the `odor_meso` repository folder, create a `paths.init` file and a `uinit.init` file.
+
+### Step 4 - Customize `docker-compose.yml`
+
++ Under `services`, add your username to `notebook_<username>`
```

Comment:
The `username` was just a placeholder for any user to put their name, so that when they run `docker compose up -d` the container name would also include their `username`.  Is this what you normally do for multiple users on the same node?

Reply:
I removed this step since it would cause a new image to be built with `docker-compose up -d`.